plugins/org.jkiss.dbeaver.data.gis.view/src/org/jkiss/dbeaver/ui/gis/panel/GISLeafletViewer.java

```diff
@@ -126,7 +126,7 @@ public GISLeafletViewer(Composite parent, @NotNull DBDAttributeBinding[] binding
         CSSUtils.setCSSClass(composite, DBStyles.COLORED_BY_CONNECTION_TYPE);
         browserCreating = true;
         try {
-            browser = new Browser(composite, SWT.NONE);
+            browser = new Browser(composite, SWT.WEBKIT);
```

Comment:
Why? As far as I remember, it may break browser creation on some OSs

Reply:
Yeah, I'm not fond of this fix either

Reply:
Really bad idea - IE is deprecated and no longer supported by Microsoft whatsoever. There is currently an effort from Eclipse to finally make EDGE the default config, so we should avoid using IE at all times.